### PR TITLE
Desodone Hotfix 5

### PR DESCRIPTION
**Pull request by Desolane**
**General**

-fixed typo in suppressed SMG description
-fixed inhand sprite for communications backpack
-fixed inhand sprite for marine ammunition belt
-adjusted rocket name and desc
-added frag and smoke grenades to marine armory vendor
-adjusted smoke grenade name and icon state

**Insurrectionist Asteroid**

-added smoke, frag, and incendiary grenades
-adjusted guns and ammo amounts
-removed atmos fuckery machine that is canister storage
-added 2 more rocket launchers with ammo to the vault, total of 3
-added a suppressed magnum to vault safe to replace rocket ammo
-fixed minor medbay map issue of wall under a door
-added more pod landing sites for more variety
-fixed places on innie asteroid base not having gravity

**ODST Spawn**

-fixed stool sprites
-adjusted jelly donut description
-added windoors to leader box
-added glass doors to SOEIV pod launch bays in leader branch
-added megaphone to leader pre-briefing room
-added smoke grenades to ODST armory

### CODE REVIEW

File: code/game/turfs/simulated/wall_types.dm [hunk 1]
```diff
@@ -6,8 +6,19 @@
...
 	..(newloc, "osmium-carbide plasteel", "osmium-carbide plasteel")
+/turf/simulated/wall/strong
+	name = "wall"
+	desc = "A huge chunk of metal used to seperate rooms."
+	icon_state = "generic"
+/turf/simulated/wall/strong/New(var/newloc)
+	..(newloc, "plasteel")
 
-
+/turf/simulated/wall/stronger
```
Comment (caelaislinn) [inline code/game/turfs/simulated/wall_types.dm]: Wall stronk. No cool scifi sounding name you can come up with here?

File: code/modules/halo/clothing/marine.dm [hunk 2]
```diff
@@ -71,8 +71,7 @@
 /obj/item/weapon/storage/belt/marine_ammo
 	name = "Ammunition Storage Belt"
 	desc = "A belt with many various pouches to hold ammunition"
-	icon = ITEM_INHAND
-	icon_override = MARINE_OVERRIDE
```
Comment (caelaislinn) [inline code/modules/halo/clothing/marine.dm:75]: Keep using these macros please. If the icon state is missing from the icon then you should modify the icon.

**Review by caelaislinn — CHANGES_REQUESTED** [review]
icon path macros

File: code/modules/halo/clothing/marine.dm (hunk 2 above)
Comment (Desolane) [reply code/modules/halo/clothing/marine.dm:75]: I couldn't get the inhand to show correctly with the marine override and when it did show inhand it didn't want to show on the back slot.

**Pull request merged**